packages/jest-console/src/getConsoleOutput.ts

```diff
@@ -40,12 +43,12 @@ export default (
     if (type === 'warn') {
       message = chalk.yellow(message);
       typeMessage = chalk.yellow(typeMessage);
-      noStackTrace = false;
+      noStackTrace = false || globalConfig.noStackTrace;
```

Comment:
`false ||` will always fallback

Reply:
Thanks for catching! updated in a following commit.